packages/world-modules/test/KeysWithValueModule.t.sol

```diff
@@ -105,6 +105,16 @@ contract KeysWithValueModuleTest is Test, GasReporter {
   }
 
   function testInstallTwice() public {
+    // Register source table
+    world.registerTable(
+      sourceTableId,
+      sourceTableFieldLayout,
+      sourceTableKeySchema,
+      sourceTableSchema,
+      new string[](1),
+      new string[](1)
+    );
```

Comment:
I assume before this was added, the test was failing because of the new reverts?

Reply:
Yes! The test was setup incorrectly by not registering the table - this catches that